doc-Managing_Providers/topics/Automation_Management_Providers.adoc

```diff
@@ -1,30 +1,42 @@
 [[automation_management_providers]]
 = Automation Management Providers
 
+[NOTE]
+====
+{product-title} ships with the Ansible role disabled. Navigate to menu:settings menu[Configuration > Settings] and select the desired server under *Zones*. Set the *Server Role* for *Embedded Ansible* to *On* to enable the role.
```

Comment:
"Navigate to menu:settings menu[Configuration > Settings] and select the desired server under *Zones* > As the settings menu is not an actual UI element, we write it in the following form, and not include it in "menu:". Example: "Navigate to the settings menu, then Configuration → Access Control → Groups to administer groups."